modeline/net/net.lisp

```diff
@@ -135,5 +148,39 @@ For the second case rescans route table every minute."
 	  (list (/ x 1e3) "k")))
     (setq dn (kbmb (car net) 0.1)
 	  up (kbmb (cadr net) 0.1))
-    (format nil "~A: ~5,2F~A/~5,2F~A " (net-device)
+    (format nil "~5,2F~A/~5,2F~A "
 	    (car dn) (cadr dn) (car up) (cadr up))))
+
+(defun fmt-ipv4 ()
+   (if *net-ipv4*
```

Comment:
:thought_balloon: A more succinct way to write the (if x x ..) is to use OR. ie.

```
(defun fmt-ipv4 ()
  (or *net-ipv4* "noip"))
```